### PR TITLE
Basic layout for detail pages

### PR DESCRIPTION
**Pull request by torhector2**
Creates a basic layout for detail pages.

Desktop
image

Mobile
image

### CODE REVIEW

**Comment by rafinskipg** [comment]
@torhector2 mhh i don't know if we need a generic layout for detail pages, is this repeated through the app? 

otherwise i think just define the styles on the delegate page

**Comment by torhector2** [comment]
@rafinskipg there was this ticket where it was suggested that
https://app.shortcut.com/dux-makerdao/story/1991/create-a-layout-component-for-the-different-profile-pages

File: apps/webapp/src/modules/layout/components/detail-layout/LeftSection.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { Box, SystemStyleObject } from '@chakra-ui/react';
+
+interface LeftSectionProps {
+  children: React.ReactElement;
+  sx?: SystemStyleObject;
+}
+
+export function LeftSection({ children, sx }: LeftSectionProps): React.ReactElement {
+  return (
```
Comment (rafinskipg) [inline apps/webapp/src/modules/layout/components/detail-layout/LeftSection.tsx:9]: i don't think we need to create this component, is just a div, better just leave it on the detaillayout

File: apps/webapp/src/modules/layout/components/detail-layout/MainSection.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { Box, SystemStyleObject } from '@chakra-ui/react';
+
+interface MainSectionProps {
+  children: React.ReactElement;
+  sx?: SystemStyleObject;
+}
+
+export function MainSection({ children, sx }: MainSectionProps): React.ReactElement {
+  return (
+    <Box as="main" overflowY="auto" sx={sx}>
```
Comment (rafinskipg) [inline apps/webapp/src/modules/layout/components/detail-layout/MainSection.tsx:10]: i don't think we need to create this component, is just a div, better just leave it on the detaillayout

**Review by rafinskipg — CHANGES_REQUESTED** [review]

File: apps/webapp/src/modules/layout/components/detail-layout/MainSection.tsx (hunk 2 above)
Comment (torhector2) [reply apps/webapp/src/modules/layout/components/detail-layout/MainSection.tsx]: @rafinskipg made those changes

**Review by rafinskipg — APPROVED** [review]

**Pull request merged**